src/app/server/Dnssd.h

```diff
@@ -79,8 +80,11 @@ class DLL_EXPORT DnssdServer
 
     /// (Re-)starts the Dnssd server
     /// - if device has not yet been commissioned, then commissioning mode will show as enabled (CM=1, AC=0)
```

Comment:
```suggestion
    /// - if device has not yet been commissioned, then commissioning mode will show as enabled (CM=1)
```

AC has gone away. We now have CM=1 and CM=2